src/profile/v100/address/wrap_ret_address.rs

```diff
@@ -87,6 +87,14 @@ macro_rules! decl_ret_wrapped_address {
                 }
             }
 
+            #[cfg(test)]
+            impl From<&str> for [< $address_type:camel Address >] {
+                /// TEST ONLY
+                fn from(value: &str) -> Self {
+                    value.parse().expect(&format!("Test failed since the passed in str is not a valid address: '{}'", value))
```

Comment:
TEST ONLY! This is for ALL our Address types. Greatly simplifies creation from `str` in tests :)